dockerfiles/dashboard_nginx/etc/nginx/conf.d/archivematica.conf

```diff
@@ -9,6 +9,7 @@ server {
   proxy_redirect off;
   proxy_buffering off;
   proxy_next_upstream error;
+  proxy_read_timeout 172800s;
```

Comment:
Can we get a comment in the file explain why we're setting this timeout, and what behaviour we were seeing before we set it? (And how to know if it's still configured incorrectly.)

Reply:
Sure :)